.github/workflows/testing.yml

```diff
@@ -40,7 +40,7 @@ jobs:
       with:
         python-version: ${{ matrix.python }}
     - name: Install Python dependencies
-      run: python -m pip install --progress-bar off --upgrade "tox<4" codecov
+      run: python -m pip install --progress-bar off --upgrade "tox"
```

Comment:
...since `codecov` appears to be no longer on PyPI after having been deprecated.

It appears that the GitHub Action for codecov handles everything for the coverage report, so there's no need to `pip install codecov` anyway.